library/build.gradle

```diff
@@ -23,7 +23,9 @@ dependencies {
 
 android {
     compileSdkVersion 23
-    buildToolsVersion "23.0.0"
+    buildToolsVersion "23.0.2"
```

Comment:
You also need to bump `ANDROID_BUILD_TOOLS_VERSION` to match in `.travis.yml` - https://github.com/googlemaps/android-maps-utils/blob/master/.travis.yml#L15.